src/converter/one/jfr/Dictionary.java

```diff
@@ -37,10 +37,6 @@ public int size() {
     }
 
     public void put(long key, T value) {
-        if (key == 0) {
```

Comment:
@apangin Is there a reason not to allows this? I've changed the JFR writer to use `Index`, so now `key=0` can happen

Reply:
`0` is used as a marker for empty slots. `keys[]` array is initially filled with zeros.